nucypher/cli/literature.py

```diff
@@ -424,6 +426,12 @@
 # Worklock
```

Comment:
Definitely not in this PR, but I'm starting to think that this `literature` file will continue to increase in size over time, and I'm wondering whether it makes sense to organize it differently. Examples of what could be done include:
1. If keeping constants all in one file, perhaps using a naming convention (prefix) for the constants eg. `WORKLOCK_...` for worklock constants, or `URSULA_...` for Ursula constants? We already segment the file into sections based on usage via comments. Common constants don't necessarily need a prefix.
2. Perhaps splitting the constants across different literature files based on how constants are used eg. `cli/literature/worklock_literature.py` for worklock constants, analogous names for other usage.

(^ @KPrasch, @cygnusv )

Reply:
Opened #2061